components/claim/admin-view-claims.tsx

```diff
@@ -86,8 +82,12 @@ const PsyIcon = () => {
 
 const AdminViewClaims = () => {
   const { previousStep, nextStep } = useWizard();
-  //  remove showEmptyState when done
-  const [showEmptyState, setShowEmptyState] = useState(false);
+  const { claims } = useGetBatchClaims();
+
+  const formatClaimAmount = (amount: string) => {
+    const formattedAmount = Number(amount) / 1e18;
```

Comment:
can we use here formatUnits?

Reply:
so claim.amount is a string that looks like this: "6744757357575800000000"

this just formats it into a readable number like 6744,76

there is no bigint involved